test/remote/gateways/remote_beanstream_test.rb

```diff
@@ -267,7 +267,7 @@ def test_successful_check_purchase_and_refund
     assert_success purchase
 
     assert refund = @gateway.refund(@amount, purchase.authorization)
-    assert_success credit
+    assert_success refund
```

Comment:
I'm actually guessing here. `credit` isn't defined, so the author probably meant `refund`, but that implies this test was never run, so…not 100% sure.